File: baekjoon/js/7567.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

let result = 10;
let prev = buf[0];

for (let i = 1; i < buf.length; i++) {
  result += buf[i] === prev ? 5 : 10;
  prev = buf[i];
}

console.log(result);

File: baekjoon/rust/7567.rs
use std::io::stdin;

fn main() {
    let mut buf = String::new();
    stdin().read_line(&mut buf).unwrap();

    let dishes: Vec<char> = buf.trim().chars().collect();
    let mut result = 10;

    for i in 1..dishes.len() {
        result += if dishes[i - 1] == dishes[i] { 5 } else { 10 };
    }

    print!("{}", result);
}
